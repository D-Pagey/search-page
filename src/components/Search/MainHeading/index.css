.main-heading-container {
  padding: 15px 0;
}

.main-heading {
  line-height: 1.1;
  margin: 0;
}