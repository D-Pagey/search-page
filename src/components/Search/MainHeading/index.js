import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const MainHeadingWrapper = styled.div`
  padding: 15px 0;
`;

const MainHeadingTitle = styled.h2`
  line-height: 1.1;
  margin: 0;
`;

export default function MainHeading({ total }) {
  return (
    <MainHeadingWrapper>
      <MainHeadingTitle>
        {total} VEHICLES FOUND NEAR LONDON, UNITED KINGDOM
      </MainHeadingTitle>
    </MainHeadingWrapper>
  );
}

MainHeading.propTypes = {
  total: PropTypes.number,
};

MainHeading.defaultProps = {
  total: '',
};
