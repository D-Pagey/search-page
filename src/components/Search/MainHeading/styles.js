import styled from 'styled-components';

export const MainHeadingWrapper = styled.div`
  padding: 15px 0;
`;

export const MainHeadingTitle = styled.h2`
  line-height: 1.1;
  margin: 0;
`;
