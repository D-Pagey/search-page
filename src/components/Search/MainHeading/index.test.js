import React from 'react';
import { shallow } from 'enzyme';
import MainHeading from '.';

it('should render', () => {
  const wrapper = shallow(<MainHeading />);
  expect(wrapper).toMatchSnapshot();
});
