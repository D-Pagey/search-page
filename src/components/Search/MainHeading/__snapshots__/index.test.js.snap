// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="main-heading-container"
>
  <h2
    className="main-heading"
  >
    0
     VEHICLES FOUND NEAR LONDON, UNITED KINGDOM
  </h2>
</div>
`;
